app/api/dao/organization.py

```diff
@@ -88,6 +70,89 @@ def update_organization(data):
             return update(organization, data, messages.ORGANIZATION_SUCCESSFULLY_UPDATED, HTTPStatus.OK)     
         return messages.NOT_ORGANIZATION_REPRESENTATIVE, HTTPStatus.FORBIDDEN
 
+    
+    @staticmethod
+    def list_organizations(
+        search_name,
```

Comment:
rename search_name to organization_name

Reply:
I have different opinion with you on this, @meenakshi-dhanani .I don't think it's right to call it organization name since it is not the organization `name` attribute, but a search query containing the `organization name`. The same manner is used on the Mentorship System search query field for list of users GET /users (although in MS it doesn't specify that it only search based on user name)
<img width="822" alt="Screen Shot 2020-08-17 at 4 15 13 pm" src="https://user-images.githubusercontent.com/29667122/90362882-efc2c980-e0a4-11ea-9eb6-305897d2199a.png">

Reply:
for example, the search_name query can contain only partial name of the organization name. In the example below I search for any organization that has `Co` in their name. 
<img width="877" alt="Screen Shot 2020-08-17 at 4 25 33 pm" src="https://user-images.githubusercontent.com/29667122/90363708-7deb7f80-e0a6-11ea-8ef5-d1e7c844d840.png">

From this query I got the response below:

<img width="1558" alt="Screen Shot 2020-08-17 at 4 25 16 pm" src="https://user-images.githubusercontent.com/29667122/90363728-8cd23200-e0a6-11ea-8fde-0c23034f9506.png">

Reply:
Thank you for clarifying. This helps. So usually in a lot of APIs that I've built and seen, if it's a /GET /organizations?name=ABC, I would expect a full text match, ie. if you search for ABC, you get ABC, if you search for AB you get no result. 
For search, we usually do a separate API eg. /organizations/search?name=AB would return ABC for a search term as "AB". 

why search would be a separate API, is because searches are very complex, because it usually is a lot of search terms, filters, improvements, etc. 

Can I ask how are we going to be using this endpoint in the UI? It would help understand what we need right now

Reply:
I'm thinking of having a filter on the organisations list table (as future feature, not now) where user can type in the query in a search field that will change the content of the table rows.

Reply:
But I've just changed and pushed to the repo with search_name replaced with name as you suggested. I think we'll keep it like that for now.

Reply:
That's ok. I've just refactored the organization response model and pushed the changes. I've changed the `name` back to `search_name` on the query. I haven't applied the changes to the tests which is why travis build fails.

Reply:
It still shows as search_name instead of name. I confirmed this with my colleagues and asked them, they had the following views

/organizations/{name} - path variable eg. /organizations/ABC = returns ABC
/organizations?name="AB" returns ABC 

I think it makes sense too. No need to put search in there. To differentiate full match we could always use path variables. 

And when we have more search terms we could think of a separate API too

Reply:
Done, changed it again to name only.